### PR TITLE
feat: allow static configurations without calling shogun

### PR DESCRIPTION
**Pull request by simonseyock**
This PR introduces two new config parameters, that allow to load the application configuration and the layers from other URLs. This way the client can be used without a shogun backend.

The config parameters are:

* `staticAppConfigUrl`
* `layerConfigUrl`

### CODE REVIEW

File: src/bootstrap.tsx [hunk 1]
```diff
@@ -348,7 +372,17 @@ const setupSHOGunMap = async (application: Application) => {
 
   view.setConstrainResolution(true);
 
-  const layers = await parser.parseLayerTree(application, projection);
+  let layers;
+
```
Comment (marcjansen) [inline src/bootstrap.tsx:376]:
```suggestion
```

File: src/bootstrap.tsx [hunk 2]
```diff
@@ -348,7 +372,17 @@ const setupSHOGunMap = async (application: Application) => {
...
+  console.log('test')
```
Comment (marcjansen) [inline src/bootstrap.tsx:377]:
```suggestion
```

**Review by marcjansen — COMMENTED** [review]
Just a stray (?) `console.log`. Not a real review.

**Review by ahennr — APPROVED** [review]

File: src/i18n/translations.ts [hunk 3]
```diff
@@ -162,6 +162,7 @@ export default {
         errorDescription: 'Aufgrund eines unerwarteten Fehlers konnte die Applikation nicht geladen werden.',
         errorDescriptionAppIdNotSet: 'Keine Applikations-ID angegeben. Bitte geben Sie die ID als Abfrageparameter an, z.B. ?applicationId=1909',
         errorDescriptionAppConfigNotFound: 'Die Applikation mit der ID {{applicationId}} konnte nicht geladen werden.',
+        errorDescriptionAppConfigStaticNotFound: 'Die Applikations Konfiguration konnte nicht geladen werden.',
```
Comment (dnlkoch) [inline src/i18n/translations.ts:165]:
```suggestion
        errorDescriptionAppConfigStaticNotFound: 'Die Applikationskonfiguration konnte nicht geladen werden.',
```

Comment (dnlkoch) [inline src/i18n/translations.ts:165]: Do we also need the translation for `en`?

**Pull request merged**